Makefile.release

```diff
@@ -9,11 +9,9 @@ __tag-release-candidate:
 	CS_VERSION="$(CS_VERSION)" ./scripts/tag-release-candidate.sh
 
 .PHONY: __build-release-candidate
-__build-release-candidate: __tag-release-candidate
+__build-release-candidate:
 	@echo "+++ Building release candidate"
 	@$(eval GCR_PREFIX := "config-management-release")
-	@test -n "$(CS_VERSION)" || (echo "CS_VERSION unset" ; exit 1)
-	$(eval RC_TAG != git tag --points-at "HEAD" --list "$(CS_VERSION)-rc.*" | sort -V | tail -n 1)
 	@test -n "$(RC_TAG)" || (echo "RC_TAG unset" ; exit 1)
```

Comment:
Is `RC_TAG` is used?

Reply:
Yes it is, if you expand the diff below a little you can see that it is used to tag the images/manifests/binaries (as this target builds RC artifacts).

Before the assumption was that the `CS_VERSION` is provided and this target would invoke the tag target to create the `RC_TAG` using `CS_VERSION`. This PR changes the assumption so that the tag target is invoked separately and that `RC_TAG` is provided to the build target. This makes it such that the build target can be easily run separately from the tag target via a tag-based trigger